pyaedt/aedt_logger.py

```diff
@@ -590,9 +591,8 @@ def _log_on_dekstop(self, message_type, message_text, level=None, proj_name=None
                 des_name = des_name[des_name.find(";") + 1 :]
             try:
                 self._desktop.AddMessage(proj_name, des_name, message_type, message_text)
-
             except Exception:  # pragma: no cover
-                pass
+                self._global.info("Failed in adding desktop message")
```

Comment:
```suggestion
                self._global.info("Failed in adding desktop message.")
```
Conclude with period.